test/acceptance/service-fabrik-api.instances.director.spec.js

```diff
@@ -951,7 +971,7 @@ describe('service-fabrik-api', function () {
               mocks.verify();
             });
         });
-
+        /////
```

Comment:
unnecessary line

Reply:
not sure how so many of them came. Removing them.